ext/zip/php_zip_arginfo.h

```diff
@@ -1,5 +1,5 @@
 /* This is a generated file, edit the .stub.php file instead.
- * Stub hash: 93db1517143e903f01e4726a6b6c9bcbe0510f5b */
```

Comment:
the change in the stub file was not committed :)

Reply:
opps, added